GH-109190: Copyedit 3.12 What's New: PEP 669 (#109658)
python/cpython: Doc/whatsnew/3.12.rst

PEP 669: Low impact monitoring for CPython
------------------------------------------

:pep:`669` defines a new :mod:`API <sys.monitoring>` for profilers,
debuggers, and other tools to monitor events in CPython.

> This instrumentation covers a wide range of events, including calls,

Reviewer: Can we avoid "instrumentation". It is how it works, but it isn't (mostly) visible in the API and might hypothetically be changed in the future. 
How about "PEP 669 supports monitoring of a wide range ..."

> This instrumentation covers a wide range of events, including calls,

Reviewer 2: ```suggestion
It covers a wide range of events, including calls,
```

returns, lines, exceptions, jumps, and more.
This means that you only pay for what you use, providing support
for near-zero overhead debuggers and coverage tools.
See  :mod:`sys.monitoring` for details.

> (Contributed by Mark Shannon in :gh:`103083`)

Reviewer 3: ```suggestion
(Contributed by Mark Shannon in :gh:`103083`.)
```
